plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -15,6 +15,7 @@
 package io.trino.plugin.faker;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
```

Comment:
> Add missing unit tests on Faker queries

The commit message is misleading when it changes the product code.

Reply:
Went with it, because unit tests forced the fixes on product code. 

Do you have any suggestions?

Reply:
The commit message should say what behavior was fixed. If there are any additional tests, you could extract them into a separate commit. This could make reviewing easier, but it's optional.

I assume you started with the intention of only adding tests, and this is expressed in the current commit message. Then noticed the code needs to be fixed, so you did that to make the tests passed. You should update the commit message to reflect the end result.

Reply:
Based on the discussion, I think it better to extract it to separate commits. Closing this issue for now.